spec/post_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  user = User.create(name: 'Alphonce', photo: 'https:/photo.com', bio: 'Teacher', posts_counter: 5)
+  subject { Post.new(author: user, title: 'Chinua', commentscounter: 10, likescounter: 7) }
+
```

Comment:
Please add tests for `#update_user_posts_count` and `#recent_comments` in Post model specs as any custom methods in the models should have corresponding unit tests.